guide/docs/popular-topics/reactions.mdx

```diff
@@ -1,8 +1,303 @@
 ---
-description: Create polls, paginate your commands, and more.
+description: This section covers the topic of interacting with message reactions. Both adding them with code, and reacting to on_reaction events
 hide_table_of_contents: true
 ---
 
 # Reactions
 
-<WorkInProgress />
+Reactions is discords way of adding emojis to other messages. Early on, before discord introduced components; this system was largely used to make interactive messages and apps.  
+Having bots react to message reactions is less common now, and is somewhat considered legacy behaviour.  
+This guide will teach you the basics of how they work, since they still have their use cases.  
+Mainly reaction role systems and polling, but there are some other uses.
+
+In Disnake reactions are represented with <DocsLink reference="disnake.Reaction">Reaction</DocsLink> objects. Whenever you operate on a <DocsLink reference="disnake.Message">Message</DocsLink> you can access a list of reactions attached to that message.
+In this guide we will be providing an example using the <DocsLink reference="disnake.on_raw_reaction_add">on_raw_reaction_add / remove</DocsLink> events and a message_command's <DocsLink reference="disnake.MessageCommandInteraction">interaction</DocsLink> to demonstrate.
+
+:::info
+**Reaction limitations**
+
+-   Removing reactions that are not owned by the bot requires <DocsLink reference="disnake.Intents.reactions">Intents.reactions</DocsLink> to be set
```

Comment:
Mentioned removing here since its practically the only action you can perform on non-owned reactions. But i will rewrite this to talk more about the cache. Also added that it is required for event listening

Reply:
Ah, that makes makes sense. Looks good, ty!